From d3fd52a8e1241bace8d149ef31f6404dd61f89df Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 22 May 2018 08:09:28 +0100
Subject: [PATCH] v2v: Use Std_utils.qemu_input_filename instead of prefixing
 "file:" to filename (RHBZ#1580292).

This also allows us to use virt-v2v with the old version of qemu in
RHEL 7.

Fixes commit e29296cfa20dd691995832940a30fe2e6b98149a.

Thanks: Pino Toscano for suggesting the fix.
(cherry picked from commit 0c934144eb8ea0ee1e1f68d4975415c001a62556)
---
 v2v/v2v.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/v2v/v2v.ml b/v2v/v2v.ml
index 90173b018..6b9bdadf6 100644
--- a/v2v/v2v.ml
+++ b/v2v/v2v.ml
@@ -726,7 +726,7 @@ and copy_targets cmdline targets input output =
       let cmd =
         let filename =
           match t.target_file with
-          | TargetFile filename -> "file:" ^ filename
+          | TargetFile filename -> qemu_input_filename filename
           | TargetURI uri -> uri in
         [ "qemu-img"; "convert" ] @
         (if not (quiet ()) then [ "-p" ] else []) @
-- 
2.17.2

